### DISCUSSION TITLE
Discount pot

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#90] [General]
I have a regular event attendee who wants to pay £20 so they have credit to book future events. The events vary in cost from £3 to £7, and they don't know in advance which events they might want to attend so I can't make suitable discount codes. Is it possible to have a discount code which allows me to add credit of £20 onto it, and the discount amount will be the full cost of any event they book, and then the discount code still has the remaining credit?  So I would need to be able to specify the total credit available over an unspecified number of bookings, rather than Value and Max Usage.

**Comment by liedekef** [comment]
That would require a discount type of code and your own code+db behind it to keep track of that ...

**Reply by TimB111** [reply]
That's a shame that it requires code and db. It seems like a simple feature that would benefit many event organisers.

**Reply by liedekef** [reply]
While I can code that up, it is not something currently in EME. And so the discount type code + php/db is currently your only option.

**Reply by TimB111** [reply]
Ok thank you. It would be great if this could be added in future.

**Reply by liedekef** [reply]
Feel free to open a feature request, but you're the first one ever asking for this, and then these things don't get implemented quickly (since it is not a 5-minute job to implement either) :-)
